README.md

```diff
@@ -485,6 +485,15 @@ servers:
 | format    | `string` | Format of files, such as `parquet`, `delta`, `json`, `csv`                                                       |
 | delimiter | `string` | (Only for format = `json`), how multiple json documents are delimited within one file, e.g., `new_line`, `array` |
 
+#### AWS Kinesis Data Streams Server Object
+
+| Field  | Type     | Description                                                               |
+|--------|----------|---------------------------------------------------------------------------|
+| type   | `string` | `kinesis`                                                                 |
+| name   | `string` | The name of the Kinesis data stream.                                      |
```

Comment:
For Kafka, we use `topic` as a field.
For Kinesis `stream` would be the equivalent and it is more specific than just `name`.